javascript-bank/bank.js

```diff
@@ -0,0 +1,44 @@
+/* exported Bank */
+
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+}
+
+// check getBalance if it's positive integer
+// create object
+// increment balance
+// add newAccount to accounts
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if (Number.isInteger(balance) === true && balance > 0) {
+    var newAccount = new Account(this.nextAccountNumber, holder);
+    Account.getBalance += newAccount.deposit(balance);
```

Comment:
Why are you trying to add/concatenate onto a function definition? Does this pass all the tests? I would think this would ruin `getBalance`